.kokoro/verify-docker-images.sh

```diff
@@ -30,9 +35,10 @@ if [[ "${branchName}" == *"release-please--branches--main"* ]] && [[ ! $gitCommi
     gcloud container images describe "${fullContainerName}" > /dev/null; exit_status=$?
     if [[ $exit_status = 0 ]]; then
       echo "Success. Found $fullContainerName."
+    else
+      exit $exit_status
```

Comment:
Wouldn't `set -o pipefail` exit with the non-zero status code? I might be missing something, but Is this check needed?

Reply:
That's a good question! Just tried it out in [f48fb2d](https://github.com/googleapis/java-shared-config/pull/689/commits/f48fb2d935c22b04c957b695f1b519cd700c8985) but unfortunately, it looks like we still need the check otherwise the script results in an exit 0 even when the command results in an error:
```
ERROR: (gcloud.container.images.describe) [:latest] digest must be of the form "sha256:<digest>".
+ exit_status=1
cleanup

[ID: 8899040] Command finished after 23 secs, exit value: 0
```